blocks/library/image/editor.scss

```diff
@@ -83,3 +83,8 @@
 		max-width: 370px !important;
 	}
 }
+
+.editor-block-list__block[data-type="core/image"][data-align="center"][data-resized="true"] .wp-block-image {
```

Comment:
Do we need the `[data-resized="true"]` seems like this style should be applied to all centered images, resized or not.

Also, not specific to this PR as not an easy fix but we should avoid relying on `.editor-block-list__block[data-type=*]` because these are not always available depending on the context of the block. For example when you save a centered block as reusable, the centering behavior might not apply properly when editing the reusable block.

Reply:
Good point on both.

The 2nd is not an easy fix, you say. Can you elaborate a bit on what situations we might end up with?

Reply:
> The 2nd is not an easy fix, you say. Can you elaborate a bit on what situations we might end up with?

It needs some reflexion. We do render the blocks in several ways: The regular visual editor (which means the `.editor-block-list__block` exists), and the edit form of a reusable block (where the parent `.editor-block-list__block` doesn't exists) . In general we should think of `BlockEdit` as a standalone component not requiring any parent.

So, we'd have to answer questions like this:

 - Should `.editor-block-list__block` be always available when rendering the visual edit mode of a block?  In which cas we'd have to refactor the code a bit.
 - Or should the block be more standalone that it is right now. in which case the `getEditWrapperProps` is problematic because it touches a potential parent div that may or may not exist and we rely on it to style the block properly.